lib/remote/apilistener-filesync.cpp

```diff
@@ -312,7 +312,15 @@ Value ApiListener::ConfigUpdateHandler(const MessageOrigin::Ptr& origin, const D
 		return Empty;
 	}
 
-	std::thread([origin, params]() { HandleConfigUpdate(origin, params); }).detach();
+	std::thread([origin, params]() {
+		try {
+			HandleConfigUpdate(origin, params);
+		} catch (const std::exception& e) {
+			Log(LogCritical, "ApiListener")
+				<< "Exception during HandleConfigUpdate: " << e.what();
```

Comment:
I'd replace "HandleConfigUpdate" w/ something more human readable.

Reply:
Updated the message.